source/_integrations/rpi_gpio.markdown

```diff
@@ -13,6 +13,10 @@ ha_domain: rpi_gpio
 
 The `rpi_gpio` integration is the base for all related GPIO platforms in Home Assistant. There is no setup needed for the integration itself, for the platforms please check their corresponding pages.
 
+## Prerequisites
+
+* 32-bit installation is required for GPIO support
```

Comment:
This depends on your platform. You are referring to HassOS specific. However, there are 64 bits platforms that can use the rpi GPIO.

Reply:
Do you prefer that I state that it's a HassOS specific requisite in the section title `## Prerequisites on HassOS` or in the item itself `32-bit installation is required for GPIO support only on HassOS`?

I like the latter slightly more.

Reply:
```suggestion
When you are running the Home Assistant Operating System, the 32-bits version of the OS is required in order to use this integration.
```